tests/integration.rs

```diff
@@ -290,7 +290,7 @@ try_test_program!(P111);
 try_test_program!(P112);
 test_program!(P113);
 test_program!(P114);
-try_test_program!(P115);
+test_program!(P115);
```

Comment:
It looks like the `P115.ok` is produces by a faulty implementation and also needs to be adjusted.